operator/config/crd/patches/webhook_in_keptnapps.yaml

```diff
@@ -2,6 +2,8 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
+  labels:
+    crdGroup: lifecycle.keptn.sh
```

Comment:
Why are you applying this here and not through kustomize `commonLabels`?

Reply:
leftover, the proper fix is in the operator pr